### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by stwunsch**
@maxgalli I'm accumulating here small fixes that I can fix right away.

### CODE REVIEW

**Comment by stwunsch** [comment]
@maxgalli Done with first round of review.

File: booking.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 from .utils import Count
 from .utils import Histogram
 
+from ROOT import gROOT
+gROOT.SetBatch(True)
```
Comment (maxgalli) [inline booking.py:11]: what does this do?

Comment (stwunsch) [reply booking.py:11]: this puts ROOT in batch mode, e.g., it's not firing up a gui thread

File: booking.py [hunk 2]
```diff
@@ -189,45 +189,31 @@ def __str__(self):
...
             raise TypeError(
                 'TypeError: not a list object.')
+        for selection in selections:
+            if not isinstance(selection, Selection):
+                raise TypeError(
+                   'TypeError: not a Selection object.')
+        self.selections = selections
 
     def __set_actions(self, actions):
-        if isinstance(actions, list):
```
Comment (maxgalli) [inline booking.py:213]: I don't really remember why I did it in this weird way in first place, thanks!

File: booking.py (hunk 1 above)
Comment (maxgalli) [reply booking.py:11]: aaah ok thanks

**Comment by stwunsch** [comment]
Commits merge by #14

**Pull request closed without merging**